core/module.go

```diff
@@ -34,9 +27,6 @@ type Module struct {
 	// The name of the module
 	Name string `json:"name"`
 
-	// The doc string of the module, if any
-	Description string `json:"description"`
```

Comment:
Oh I saw it was unused completely and just removed it; did not realize it was in graphql. I put it back in for now since I'm not trying to scope stuff like that in here (though it technically doesn't matter either way while it's unused); agree it may make sense to remove but can be done separately.